Clickable event card

sdotson opened the pull request:
This is addressing issue #56 . Open to other solutions! One though I had is that it might be helpful to add a hover style, perhaps a background-color, so users know that something is going to happen if they click the list item.

pdw207 commented on src/components/EventCard/EventCard.js:
```diff
@@ -25,25 +25,25 @@ const EventCard = ({ event, className }) => {
-        <div className={styles.dateLocation}>
-          <DateBlock date={startDate} />
-          { renderLocation(loc) }
+    <Link to={`/event/${_id}`}>
```
You can't have an `li` inside an `a` tag.  Just change the nesting and `ul li a { display:block; }`

InciteDemocracy replied:
@sdotson A slight darkening of the background on hover sounds like it could work nicely!

sdotson commented:
- added a hover bg color to the li
- moved anchor back to inside li
- fixed styles of li > a
- moved li padding to a (so that the whole li area would be clickable and not have that awkward 35px dead zone)

Take a look. Happy to receive further feedback.

sdotson commented:
Went ahead and fixed merge conflict.
